MediaBrowser.Controller/Entities/Folder.cs

```diff
@@ -186,22 +186,18 @@ public override bool IsVisible(User user)
         {
             if (this is ICollectionFolder && !(this is BasePluginFolder))
             {
-                var blockedMediaFolders = user.GetPreference(PreferenceKind.BlockedMediaFolders);
+                var blockedMediaFolders = user.GetPreference<Guid>(PreferenceKind.BlockedMediaFolders);
                 if (blockedMediaFolders.Length > 0)
                 {
-                    if (blockedMediaFolders.Contains(Id.ToString("N", CultureInfo.InvariantCulture), StringComparer.OrdinalIgnoreCase) ||
-
-                        // Backwards compatibility
-                        blockedMediaFolders.Contains(Name, StringComparer.OrdinalIgnoreCase))
```

Comment:
I'm not sure how this could ever be; I can revert if we think this will cause issues.

Reply:
If someone comes all the way from Emby 3.5.2?

Reply:
I looked at some 5 year old forum posts and it looks like blocked media folders was guids even then, so we're probably safe 🤷